config.md

```diff
@@ -27,13 +27,15 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
 * **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+* **`autoremove`** (bool, optional) If true then the container will be automatically deleted once the PID namespace is deleted. This removes the need for an explicit call to the `delete` operation. Defaults to false.
```

Comment:
I was assuming that the runtime could take lots of options (e.g. override for cmd) but in the end they would populate the config.json and that's all the runtime would need to do its job.

If we allowed this one flag to be special, I'm not sure where we draw the line.

But, in the end its not a big deal either way.